### PR TITLE
Add myself to OWNER file

### PR DESCRIPTION
**Pull request by JoaoBraveCoding**
Add JoaoBraveCoding to OWNERS and remove prashbnair and fpetkovski

## Description

_Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.
If it fixes a bug or resolves a feature request, be sure to link to that issue._

## Type of change

_What type of changes does your code introduce to the Prometheus operator? Put an `x` in the box that apply._

- [ ] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [ ] `FEATURE` (non-breaking change which adds functionality)
- [ ] `BUGFIX` (non-breaking change which fixes an issue)
- [ ] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [x] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

## Changelog entry

_Please put a one-line changelog entry below. This will be copied to the changelog file during the release process._

```release-note

```

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -4,27 +4,25 @@ reviewers:
...
 - simonpasquier
 - slashpai
-- prashbnair
 - arajkumar
 - sthaha
 - PhilipGough
 - bison
-- fpetkovski
 - raptorsun
 - jan--f
```
Comment (arajkumar) [inline OWNERS:12]: Can you make PhilipGough and bison as emeritus approvers?

File: OWNERS [hunk 2]
```diff
@@ -4,27 +4,21 @@ reviewers:
...
 approvers:
 - bwplotka
 - simonpasquier
 - slashpai
-- prashbnair
 - arajkumar
 - sthaha
-- PhilipGough
-- bison
-- fpetkovski
```
Comment (slashpai) [inline OWNERS:25]: We should have prashbnair and fpetkovski as as emeritus approvers

Comment (JoaoBraveCoding) [reply OWNERS:25]: Done :)

**Pull request merged**